server/utils/placementApplications/table.test.ts

```diff
@@ -15,6 +15,37 @@ describe('table', () => {
     jest.mocked(sentenceCase).mockReturnValue(stubSentenceCase)
   })
 
+  describe('placementApplicationsTable', () => {
+    it('returns the correct table', () => {
+      const tasks = placementApplicationTaskFactory.buildList(1)
+      expect(placementApplicationsTable(tasks, 'person', 'asc', '/')).toEqual({
+        firstCellIsHeader: true,
+        head: [
+          {
+            attributes: {
+              'aria-sort': 'ascending',
+              'data-cy-sort-field': 'person',
+            },
+            html: '<a href="/?undefined">Name</a>',
+          },
```

Comment:
Might be better replacing this with `sortHeader<TaskSortField>('Name', 'person', 'person', 'asc', '/'),` - not sure where that `/?undefined` is coming from?

Reply:
oops, yeah

Reply:
I've used the function directly instead of using the result of it